src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/RealTimeSpeed/realtimespeed.cpp

```diff
@@ -0,0 +1,28 @@
+#include "realtimespeed.hpp"
+
+RealTimeSpeed::RealTimeSpeed(int* Asindex, double* Astack, int* Bsindex, double* Bstack){
+    sindex=0;
+	lastspeed=0.0;
+	difference=SENSOR_DISTANCE;
+	sstack[0]=Astack;
+	sstack[1]=Bstack;
+	ssindex[0]=Asindex;
+	ssindex[1]=Bsindex;
+}
+bool RealTimeSpeed::Update(){
+	if((sindex<*ssindex[0])&&(sindex<*ssindex[1])){

```

Comment:
Same unstructured mess again without any comments detailing what is happening there.